src/utils/add.trust.ts

```diff
@@ -45,13 +46,15 @@ const getPageProperties = (
 
   const appData = getApplicationData(req.session);
 
+  // note: isUpdate covers both Update and Remove journeys, so when on Remove journey isUpdate will be true.
```

Comment:
Is it a big refactor job to rename this field/variable to `isUpdateOrRemove`?

Reply:
there will be a couple of templates to update and their associated unit tests. Also, isUpdate is used widely throughout the controllers, it might seem odd that one has an isUpdateOrRemove when others all have isUpdate

Reply:
A bit more tech debt then...